packages/@ember/-internals/utils/lib/mandatory-setter.ts

```diff
@@ -1,8 +1,12 @@
 import { assert } from '@ember/debug';
+import { _WeakSet as WeakSet } from '@ember/polyfills';
 import { DEBUG } from '@glimmer/env';
+import { Tag } from '@glimmer/validator';
```

Comment:
I don't think `@glimmer/validator` exists in this branch (this fails the linting)

Reply:
Ah, yeah, strange that tests are passing (I ran them locally)

Reply:
It’s because it’s only used for types, not runtime

Reply:
Rebased and fixed the import, verified linting passes...